examples/eks/eks_cluster_live_migration/castai.tf

```diff
@@ -0,0 +1,15 @@
+module "cluster" {
+  source = "./module/castai"
+  count  = var.enable_castai ? 1 : 0
+
+  cluster_name     = var.cluster_name
+  castai_api_token = var.castai_api_token
+  cluster_region   = var.cluster_region
+  vpc_id           = module.vpc.vpc_id
+  security_groups = [
+    module.eks.cluster_security_group_id,
+    module.eks.node_security_group_id,
+    aws_security_group.additional.id,
+  ]
+  subnets = module.vpc.private_subnets
```

Comment:
depend_on = [ module.eks[

Reply:
Unfortuantelly this doesnt work with module.eks I get circural dependencies. with only modle.vpc I get some cryptic error with "for each` something something. So for now it is better that wat was before.